### PR TITLE
Climb speed

### PR DESCRIPTION
**Pull request by JFranklin3000**
##### In this PR
- Added Climb Speed as a movement type
- Added half swim and climb speeds if there is nothing setting them. 

##### Testing
- Add Origin to character and make sure movements are calculated correctly.

### CODE REVIEW

File: module/documents/actor.mjs [hunk 1]
```diff
@@ -171,7 +171,7 @@ export class Essence20Actor extends Actor {
     const system = this.system;
     system.movementIsReadOnly = true;
 
-    const movementTypes = ['aerial', 'ground', 'swim'];
+    const movementTypes = ['aerial', 'ground', 'climb', 'swim'];
```
Comment (phildominguez) [inline module/documents/actor.mjs:174]: ABC order

File: module/documents/actor.mjs [hunk 2]
```diff
@@ -195,8 +195,16 @@ export class Essence20Actor extends Actor {
...
+
+      if (system.movement[movementType].total == 0) {
+        if (movementType == 'climb' || movementType == 'swim') {
+          system.movement[movementType].total = system.movement.ground.total * .5;
+        }
+      }
     }
 
+
+
```
Comment (phildominguez) [inline module/documents/actor.mjs:207]:
```suggestion
```

File: lang/en.json [hunk 3]
```diff
@@ -234,6 +234,7 @@
     "ModeBotMode": "Bot Mode",
     "ModeAny": "Any Mode",
     "MovementTypeAerial": "Aerial",
+    "MovementTypeClimb": "Climb ",
```
Comment (phildominguez) [inline lang/en.json:237]:
```suggestion
    "MovementTypeClimb": "Climb",
```

File: module/documents/actor.mjs (hunk 1 above)
Comment (phildominguez) [inline module/documents/actor.mjs:174]: Should the Origins item sheet have an input for Climb?

**Review by phildominguez — CHANGES_REQUESTED** [review]

File: module/documents/actor.mjs (hunk 1 above)
Comment (JFranklin3000) [reply module/documents/actor.mjs:174]: There aren't currently any Origins that grant a climb speed so I left it off at least for now.

Comment (JFranklin3000) [reply module/documents/actor.mjs:174]: ABC order doesn't work, because ground isn't set to compare to climb then.

**Review by phildominguez — APPROVED** [review]

**Pull request merged**